stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/factory/ChannelListViewModelFactory.kt

```diff
@@ -28,7 +28,7 @@ public class ChannelListViewModelFactory @JvmOverloads constructor(
     ),
     private val sort: QuerySort<Channel> = ChannelListViewModel.DEFAULT_SORT,
     private val limit: Int = 30,
-    private val messageLimit: Int = 1,
+    private val messageLimit: Int = 3,
```

Comment:
Just curious, what other SDKs are doing n that case?

Reply:
RN: The show "message deleted", if the last message is deleted. So they don't have this problem.
Flutter: The same behaviour that we have.

Reply:
🤔  I would prefer to show "message deleted", even if theirs deleted message is not visible in the chat itself

Reply:
I think we shouldn't update our MessageLimit only because this bad design thing.
I would prefer to follow the same RN approach, show that the last message was deleted.
This approach only cover the case where some of the last three messages wasn't deleted, but it is not a solution for all cases

Reply:
this will cause a regression for performance since the last message is available in offline storage. the last 3 require N extra queries